Refactor FE deploy for latest terminology (#1630)
RoundingWell/care-ops-frontend: README.md

Reviewer: ```suggestion
Occasionally the Vite server does not release the port, so development in either environment will use the wrong port and fail.
```

#### Backend Setup

To develop against a remote AWS dev organization:

1. **Deploy your backend organization** (replace `john` with your organization slug):
   ```
   bin/console deploy:dev john
   ```

2. **After your first login**, prepare a clinician account:
   ```

>    bin/remote <organization> console clinician:prepare your.email@roundingwell.com

Reviewer: ```suggestion
   bin/remote <organization> console clinician:prepare <email>
```

   ```

>    For example, if your organization slug is `john`:

Reviewer: ```suggestion
   For example, if your organization is `john` and your email is `john.doe@roundingwell.com`:
```

   ```
   bin/remote john console clinician:prepare john.doe@roundingwell.com
   ```

#### Frontend Setup

Create a `.env` file in the project root with your dev organization URL:
```
VITE_DEV_API_URL=https://john.roundingwell.dev
```

> Replace `john` with your organization slug.

Reviewer: ```suggestion
_Note: Replace `john` with **your** organization slug._
```

### Running the Development Server

There are two npm commands most useful for development:

Useful for local development with vite local dev-server and your remote dev organization (https://github.com/RoundingWell/care-ops-backend)
```
$ npm run dev

> ```

Reviewer: These two sentences read oddly.
